html/Corporate/js/video.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef */
+document
+    .getElementById('banner__video-btn')
+    .addEventListener('click', function() {
+      let video = document.getElementById('banner__video');
+      if (video.paused === true) {
+        video.play();
+        document
+            .getElementById('banner__video-btn')
```

Comment:
wouldn't code cleaner if save dom elements to variables and after work with them and don't take all the time dom method to find this elements?

plus in this case querySelector method looks pretty well
and generally id's is not very good practice as there is no standard how browser will behave in case of duplicates